docs/source/overview/index.rst

```diff
@@ -0,0 +1,7 @@
+*********
+Overview
+*********
+
+.. mdinclude:: ../../../README.md
```

Comment:
Might it be better to rewrite README.md as README.rst? I'm not familiar with `mdinclude`, but the rendered docs you linked have at least one rendering error here:

![image](https://user-images.githubusercontent.com/5603551/122939549-b6e92600-d341-11eb-97f8-3eaf1d46154b.png)

Reply:
Updated README.md to resolve this rendering error.